koku/api/provider/provider_manager.py

```diff
@@ -160,15 +165,29 @@ def get_infrastructure_info(self):
         """Get the type/uuid of the infrastructure that the provider is running on."""
         if self.model:
             if self.model.infrastructure and self.model.infrastructure.infrastructure_type:
+                source = Sources.objects.get(koku_uuid=self.model.infrastructure.infrastructure_provider_id)
                 return {
                     "type": self.model.infrastructure.infrastructure_type,
                     "uuid": self.model.infrastructure.infrastructure_provider_id,
+                    "id": source.source_id,
+                    "account": self.model.infrastructure.infrastructure_account,
+                    "region": self.model.infrastructure.infrastructure_region,
                 }
         return {}
 
     def get_additional_context(self):
         """Returns additional context information."""
-        return self.model.additional_context if self.model else {}
+        base_additional_context = self.model.additional_context if self.model else {}
+        if self.manifest and self.model.type == self.model.PROVIDER_OCP:
+            base_additional_context["operator_version"] = self.manifest.operator_version
+            base_additional_context["operator_airgapped"] = self.manifest.operator_airgapped
+            base_additional_context["operator_certified"] = self.manifest.operator_certified
+            base_additional_context["operator_update_available"] = (
```

Comment:
**Optional NitPick:**
There is a possible readability improvement you could make here. 
```
latest_version = utils.get_latest_operator_version()
current_version = self.manifest.operator_version.split(":")[-1].lstrip("v")
base_additional_context["operator_update_available"] = current_version != latest_version
```